__tests__/infinite_test.js

```diff
@@ -407,525 +364,489 @@ describe('The Behavior of the Variable Height React Infinite Component', functio
+    rootNode.simulate('scroll', {
       target: rootDomNode
     });
 
-    expect(function() {
-      TestUtils.findRenderedDOMComponentWithClass(rootNode, 'delegate-div');
-    }).toThrow();
+    expect(rootNode.find('.delegate-div').exists()).toBe(false);
   });
 
   it('displays the loadingSpinnerDelegate when isInfiniteLoading', function() {
-    var infiniteSpy = jasmine.createSpy('infiniteSpy');
-    var elementHeight = 200;
-    var rootNode = TestUtils.renderIntoDocument(
-        <Infinite elementHeight={elementHeight}
-                  containerHeight={800}
-                  onInfiniteLoad={infiniteSpy}
-                  infiniteLoadBeginEdgeOffset={1000}
-                  loadingSpinnerDelegate={<div className={"delegate-div"} />}
-                  className={"correct-class-name"}>
-          {renderHelpers.divGenerator(20, elementHeight)}
-        </Infinite>
-      );
+    const infiniteSpy = jasmine.createSpy('infiniteSpy');
+    const elementHeight = 200;
+    const rootNode = mount(
+      <Infinite
+        elementHeight={elementHeight}
+        containerHeight={800}
+        onInfiniteLoad={infiniteSpy}
+        infiniteLoadBeginEdgeOffset={1000}
+        loadingSpinnerDelegate={<div className={'delegate-div'} />}
+        className={'correct-class-name'}
+      >
+        {renderHelpers.divGenerator(20, elementHeight)}
+      </Infinite>
+    );
 
-    var rootDomNode = ReactDOM.findDOMNode(rootNode);
+    const rootDomNode = rootNode.getDOMNode();
     rootDomNode.scrollTop = 3600;
-    TestUtils.Simulate.scroll(rootDomNode, {
+    rootNode.simulate('scroll', {
       target: rootDomNode
     });
 
-    expect(function() {
-      TestUtils.findRenderedDOMComponentWithClass(rootNode, 'delegate-div');
-    }).not.toThrow();
+    expect(rootNode.find('.delegate-div').exists()).toBe(true);
   });
 });
 
 describe("Maintaining React Infinite's internal scroll state", function() {
   it('has does not have pointer-events: none by default', function() {
-    var infiniteSpy = jasmine.createSpy('infiniteSpy');
-    var elementHeight = 200;
-    var rootNode = TestUtils.renderIntoDocument(
-        <Infinite elementHeight={elementHeight}
-                  containerHeight={800}
-                  timeScrollStateLastsForAfterUserScrolls={10000}
-                  className={"correct-class-name"}>
-          {renderHelpers.divGenerator(20, elementHeight)}
-        </Infinite>
-      );
-    var wrapper = rootNode.smoothScrollingWrapper;
-    expect(wrapper._style._values['pointer-events']).toBeUndefined();
+    const elementHeight = 200;
+    const rootNode = renderer.create(
+      <Infinite
+        elementHeight={elementHeight}
+        containerHeight={800}
+        timeScrollStateLastsForAfterUserScrolls={10000}
+        className={'correct-class-name'}
+      >
+        {renderHelpers.divGenerator(20, elementHeight)}
+      </Infinite>
+    );
+
+    expect(rootNode).toMatchSnapshot();
```

Comment:
I'm not so sure that a snapshot does a better job of capturing this than explicitly looking for `pointer-events` being undefined. The test's description is much more specific than the test itself